test/client.spec.js

```diff
@@ -431,4 +440,53 @@ describe('Vue.js JHipster blueprint', () => {
             assert.fileContent('package.json', '"vuelidate"');
         });
     });
+    describe('Common module', () => {
+        before((done) => {
+            helpers
+                .run('generator-jhipster/generators/app')
+                .withOptions({
+                    'from-cli': true,
+                    skipInstall: true,
+                    blueprint: 'vuejs',
+                    skipChecks: true
+                })
+                .withGenerators([
+                    [
```

Comment:
Now that this blueprint has multiple subgenerators, I think you should load all blueprint subgens here, and in all tests above.

E.g. since you've not loaded the vuejs `client` subgen, what you're testing [below ](https://github.com/jhipster/jhipster-vuejs/pull/197/files#diff-82102f1b0a5c38fc1856fabe6fd374d0R488) is in fact the Angular `webpack.dev.js` (because of [this](https://github.com/jhipster/generator-jhipster/blob/master/generators/client/index.js#L357))